src/SDKs/_metadata/peering_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2020-02-27 20:33:18 UTC
```

Comment:
This file should be in `/eng/mgmt/mgmtmetadata/` and seems to be in old format. Please follow the guide to [update your build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) then generate again.
Thanks